test/rules.jl

```diff
@@ -129,11 +129,9 @@ _second(t) = Base.tuple_type_head(Base.tuple_type_tail(t))
         Ω = complex_times(x)
         Ω_fwd, Ω̇ = frule((nothing, ẋ), complex_times, x)
         @test Ω_fwd == Ω
-        @test Ω̇ ≈ jvp(central_fdm(5, 1), complex_times, (x, ẋ))
```

Comment:
If we are not going to remove this whole testset, maybe we should have somethign to replace the test.
Maybe checking the type is right makes sense.

```
@test Ω̇ isa Complex
@test ∂x isa Real
```

I think that would make some sense, so we at least check that `@scalar_rule` is doing something sensible.

Or maybe we should hand calculate the solution and put it in.

```
@test Ω̇ ≈ (1 + 2im) *  ẋ
@test ∂x ≈  (1 - 2im) * Ω̄
```

I think I over thought how rteasonable it would be to remove this,
I forgot that we should have a basic check on what `@scalar_rule` does

Reply:
Hand-calculating the solution seems ok, and better than checking the type (because more specific test). I'm unsure given my lack of familiarity with the package, but happy to go with this.